### PR TITLE
fix: issue #42

### PR DESCRIPTION
**Pull request by praveeno**
show close button, on click on close clearn input text
input search bar should visible in error screen also

issue #42

### CODE REVIEW

**Comment by luizrrodrigues** [comment]
Closing this, is valid fix but was fixed in challenge part 2.

**Review by luizrrodrigues — APPROVED** [review]

**Review by luizrrodrigues — CHANGES_REQUESTED** [review]
Check feedback in issue #42

File: src/containers/Challenges/Listing/index.jsx [hunk 1]
```diff
@@ -45,19 +47,27 @@ const Listing = ({
               <img src={IconSearch} alt="search" />
             </span>
             <TextInput
-              value={search}
+              value={searchVal}
```
Comment (luizrrodrigues) [inline src/containers/Challenges/Listing/index.jsx:50]: @praveeno Please make sure this change not overwriting existing implementation

**Review by luizrrodrigues — CHANGES_REQUESTED** [review]

File: src/containers/Challenges/Listing/index.jsx (hunk 1 above)
Comment (praveeno) [reply src/containers/Challenges/Listing/index.jsx:50]: searchVal is just handling search state at component level

**Pull request closed without merging**